asset/css/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+/* Mobile View */
+@media all and (max-width: 375px) {
+  * {
+    padding-left: 5px;
+    padding-right: 5px;
+  }
```

Comment:
Your design has margins that are not present in the template design, please remove them.

<img width="486" alt="Screen Shot 2022-06-29 at 18 23 00" src="https://user-images.githubusercontent.com/82426682/176561985-7d946973-5648-445c-b145-befd29224efa.png">

Reply:
Removed